### PR TITLE
Release alt

### PR DESCRIPTION
**Pull request by AlexSP0**
This is .spec file to build rpm package for ALT Linux.

### CODE REVIEW

File: .gear/modbus-utils.spec [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Name: modbus-utils
+Version: 1.0.0
+Release: alt1
+
+Summary: CLI utilities to work with Modbus devices
+License: %mit
+
+BuildRequires(pre): rpm-build-licenses
+BuildRequires(pre): rpm-macros-cmake
```
Comment (NIR-Ginko) [inline .gear/modbus-utils.spec:11]: Please move these directives lower, before `BuildRequires`.

**Review by NIR-Ginko — CHANGES_REQUESTED** [review]

File: .gear/modbus-utils.spec (hunk 1 above)
Comment (AlexSP0) [reply .gear/modbus-utils.spec]: Fixed!

**Comment by NIR-Ginko** [comment]
Ping @Krzysztow . Please review and accept the patch if possible. We have tested the build on ALT Workstation 9 and it works fine. We are willing to package the software for ALT Linux and this is our build system instructions which may be also used by other distributions.

**Comment by Krzysztow** [comment]
Hi, sorry for delays. I'm pretty overwhelmed with things going o. Thanks for contribution.

**Pull request merged**